Okta.AspNetCore/Okta.AspNetCore.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Official Okta middleware for ASP.NET Core 2.0+. Easily add authentication and authorization to ASP.NET Core applications.</Description>
     <Copyright>(c) 2018 Okta, Inc.</Copyright>
```

Comment:
Maybe we should update this to 2019 like in https://github.com/okta/okta-aspnet/pull/84/files#diff-6dd76ea6f734baad2516d413eed0f382R5

I updated that other one when I updated aspnet to 1.1.6